tools/start_server.sh

```diff
@@ -0,0 +1,14 @@
+# script must be executed from root git directory
+
+# kill previous instance of flask server and free port if needed
+pkill -f main.py
```

Comment:
Bug prone, it's a too generic name, another script with the same name could be running.